modules/gapi/include/opencv2/gapi/media.hpp

```diff
@@ -78,6 +81,7 @@ class GAPI_EXPORTS MediaFrame::IAdapter {
     virtual ~IAdapter() = 0;
     virtual cv::GFrameDesc meta() const = 0;
     virtual MediaFrame::View access(MediaFrame::Access) = 0;
+    virtual cv::util::any blobParams() const = 0;
```

Comment:
No as it breaks all the existing adapters. Please supply a default implementation instead (and make it safe enough to call by default - so no asserts/exceptions here if _this_ default implementation is used).

Reply:
Got you, fixed